### PR TITLE
Feat/security lp

### PR DESCRIPTION
**Pull request by MildTomato**
Started using markdown files to produce landing page
Screenshot 2022-08-17 at 9 21 41 AM
Screenshot 2022-08-17 at 9 22 11 AM

### CODE REVIEW

**Review by saltcod — APPROVED** [review]

File: apps/www/pages/security.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+Sensitive information like access tokens and encryption keys are encrypted at the application level before they are stored in the database.
+
+</Section>
+
+<Section icon={<IconGitHub size={19} />}>
+
+### Github security integration
+
+[We have partnered with GitHub]('/blog') to scan for Supabase service role API keys. If Github detects any keys with full access to the database being pushed to GitHub, they are automatically revoked.
```
Comment (Isaiah-Hamilton) [inline apps/www/pages/security.mdx:78]: You don't need quotes around the link or it will break.

File: apps/www/pages/security.mdx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+</div>
+
+</Section>
+<Section icon={<KeyIcon strokeWidth={1}/>}>
+
+### Data Encryption
+
+All customer data is encrypted at REST with AES-256 and in transit via TLS.
+
+Sensitive information like access tokens and encryption keys are encrypted at the application level before they are stored in the database.
```
Comment (kiwicopple) [inline apps/www/pages/security.mdx:69]:
```suggestion
Sensitive information like access tokens and keys are encrypted at the application level before they are stored in the database.
```

File: apps/www/pages/security.mdx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+</Section>
+
+<Section icon={<IconGitHub size={19} />}>
+
+### Github security integration
+
+[We have partnered with GitHub]('/blog') to scan for Supabase service role API keys. If Github detects any keys with full access to the database being pushed to GitHub, they are automatically revoked.
```
Comment (kiwicopple) [inline apps/www/pages/security.mdx:77]:
```suggestion
[We have partnered with GitHub]('/blog') to scan for Supabase service role API keys. If any Supabase API keys are pushed to GitHub, they are automatically revoked.
```

File: apps/www/pages/security.mdx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+Members of organizations in Supabase can be granted access to specific resources.
+
+</Section>
+
+<Section icon={<RewindIcon strokeWidth={1}/>}>
+
+### Backups
+
+All customer databases are fully backed up every day.
```
Comment (kiwicopple) [inline apps/www/pages/security.mdx:93]:
```suggestion
All customer databases are backed up every day.
```

File: apps/www/pages/security.mdx [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+
+Stripe is a certified PCI Service Provider Level 1, which is the highest level of certification in the payments industry.
+
+</Section>
+
+<Section icon={<ClipboardCheckIcon strokeWidth={1}/>}>
+
+### Vulnerability Management
+
+Supabase works with industry experts to conduct annual penetration tests.
```
Comment (kiwicopple) [inline apps/www/pages/security.mdx:112]:
```suggestion
Supabase works with industry experts to conduct regular penetration tests.
```

File: apps/www/pages/security.mdx [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+In addition to internal security reviews, we use various tools to scan our code for vulnerabilities including Github, Vanta, and Snyk.
```
Comment (kiwicopple) [inline apps/www/pages/security.mdx:114]:
```suggestion
In addition to internal security reviews, we use various tools to scan our code for vulnerabilities including GitHub, Vanta, and Snyk.
```

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**